marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/data-services/mapping/getNewDocumentTest.sjs

```diff
@@ -0,0 +1,183 @@
+const test = require("/test/test-helper.xqy");
+const DataHubSingleton = require('/data-hub/5/datahub-singleton.sjs');
+const dataHub = DataHubSingleton.instance();
+const hubTest = require("/test/data-hub-test-helper.sjs");
+
+const jsonUri = '/test/sampleCustomerDoc.json';
+const jsonInput = {
+  envelope:{
+    instance:{
+      CustOrders:{
+        CustomerID:204,
+        Name:{
+          FirstName: 'Jacqueline',
+          LastName: 'Knowles'
+        },
+        Nicknames:{
+          Nickname:[
+            'Jack',
+            'Sparrow'
+          ]
+        }
+      }
+    }
+  }
+};
+
+const xmlUri = '/test/sampleCustomerDoc.xml';
+const xmlInput = `
+<Order xmlns="https://www.w3schools.com/OrderNS">
```

Comment:
I recommend slimming this down to the bare minimum needed to verify that the XML parsing is working correctly. For example, you likely don't need all of the top-level properties that do not have a namespace - those are just adding noise.

Reply:
I was able to remove some elements that were not being tested, and added some tests for other unique portions that were not otherwise tested.